pommerman/network/ionc/constants.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+"""IonClient Constants"""
+
+# Imports
+from enum import Enum as enum
+
+
+class LangEN(enum):
```

Comment:
FYI, enums are usually capitalized to represent them as being constants. I'm not going to gripe about that here, but just keep it in mind for future.